frontend/components/Map.jsx

```diff
@@ -0,0 +1,117 @@
+import 'leaflet/dist/leaflet.css'
```

Comment:
I think we've been using `tsx` over `jsx` if you're comfortable!

Reply:
Also, wondering if we might want to break these out into their own component files?